### PR TITLE
Sunset animation before switching to loader

### PR DESCRIPTION
**Pull request by CalebJohn**
I got sidetracked in an attempt to make a smoother cutscene transition and ending up adding a small animation to the menu, I hope you guys like it.

### CODE REVIEW

**Comment by clayjohn** [comment]
Looks good to me. Although it does give me an idea. We could probably do they same sort of thing that  we have here on the loading screen. When you hit start the sun could jump up to the top then slowly work its way down to the bottom as the game loads.

File: src/states/Menu.js [hunk 1]
```diff
@@ -90,6 +91,14 @@ Menu.prototype = {
     this.optionBtns.visible = false;
   },
 
+  //plays a simple animation and switches to cutsence state
+  startGame: function() {
+    this.sunPos = {y: this.sunPos.y};
+
+    var tween = game.add.tween(this.sunPos).to({y: game.height}, 2500, Phaser.Easing.Exponential.in, true);
+    tween.onComplete.add(function() {game.state.start('Cutscene');}, this);
```
Comment (ExtraGravity) [inline src/states/Menu.js:103]: Let's keep this on multiple lines.

**Comment by ExtraGravity** [comment]
I think we should keep spaces around braces, so like `{ y: 52 }` as opposed to `{y: 52}`

File: src/states/Menu.js (hunk 1 above)
Comment (CalebJohn) [inline src/states/Menu.js:103]: Split it into multiple lines? Or is it fine now?

**Comment by CalebJohn** [comment]
@clayjohn that was the original intention for this pull request but it would likely require some restructuring, I'll be looking into it again tonight.

@enochtsang the motivation for extra spaces is for readability, correct? Currently the micro-style guide that we have in the readme specifies no extra spaces, but we can change it if we all agree.

File: src/states/Menu.js (hunk 1 above)
Comment (ExtraGravity) [inline src/states/Menu.js:103]: We should change it to be split into multiple line\* sorry for the confusion ;o

**Comment by ExtraGravity** [comment]
Just tested it, looks good! But I think it should be a bit faster. As to your last comment, if it's per the style guide then it's fine, we can keep it that way.

**Comment by CalebJohn** [comment]
Alright, this integrates the changed that @enochtsang mentioned, let me know if it's not fast enough.

edit: Lets holdoff on merging this as I'm working on another branch that will supersede this.

**Comment by CalebJohn** [comment]
Looks like this will be covered by #49

**Pull request closed without merging**